pontoon/sync/tests/test_changeset.py

```diff
@@ -300,3 +301,41 @@ def test_obsolete_db(self):
         self.changeset.execute()
         self.main_db_entity.refresh_from_db()
         assert_true(self.main_db_entity.obsolete)
+
+    def test_multiple_locale_authors(self):
+        """
+        Test if authors are allocated per locale properly.
+        Regression tests against bug 1214722.
+
+        Contributors should map to locales following way:
+            first_locale -> first_user, third_user
+            second_locale -> second_user
+        """
+        first_entity, second_entity = EntityFactory.create_batch(2)
```

Comment:
You already have two entities and locales available from the setup method, no need to create new ones. Same goes for the resources and entities being created below. You should only need to create the translations to test what you want.